app/controllers/v1/political_parties_controller.rb

```diff
@@ -23,4 +31,23 @@ def set_party
     @party = PoliticalParty.find_by(id: params[:id])
     render status: :not_found && return unless @party
```

Comment:
Is this method support to render the @party as a json object? I can see it returns 404 if @party is not present but what happens if it is?